input/resources/au-core-server.xml

```diff
@@ -236,42 +236,6 @@
 </tr>
```

Comment:
Fix "http://hl7.org/fhir/R4/composition.html#search" to be "http://hl7.org/fhir/R4/medicationrequest.html#search" - location of this comment is not related to where this fix needs to be applied

Reply:
Change applied (not pushed yet).